src/server/controllers/events.controller.js

```diff
@@ -4,6 +4,14 @@ const omit = require('lodash/omit');
 const eventsService = require('../services/events.service');
 const userService = require('../services/user.service');
 
+function isAuthenticated(request, response, next) {
```

Comment:
I'll use `checkPermissions` from `src/server/utils/security-utils.js`. It already exist and it checks authorization. `isAuthenticated` function from that wouldn't work properly.